theme/_layouts/default.html

```diff
@@ -1,24 +1,23 @@
 <!DOCTYPE html>
+<html lang="en">
 <html>
-
   {% include head.html %}
 
+  {% include nav.html %}
   <body>
 
-    {% include header.html %}
-
-    <div>
+<div>
       <h1 class="display-4 text-center mt-3">{{ page.title }}</h1>
```

Comment:
Wondering if these titles are a little big? Maybe we should go to `display-2` instead?